deis/manifests/deis-builder-rc.yaml

```diff
@@ -16,8 +16,8 @@ spec:
     spec:
       containers:
         - name: deis-builder
-          image: quay.io/deisci/builder:v2-alpha
-          imagePullPolicy: Always
+          image: quay.io/deis/builder:v2-alpha
+          imagePullPolicy: IfNotPresent
```

Comment:
Why change the policy? Should alpha stay up to date with the alpha channel or do we always want full teardowns to get new image?